clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -337,7 +345,7 @@ public void testFetchSkipsBlackedOutNodes() {
 
         assignFromUser(singleton(tp0));
         subscriptions.seek(tp0, 0);
-        client.updateMetadata(initialUpdateResponse);
+        client.updateMetadata(initialUpdateResponseWithLeaderEpoch);
```

Comment:
Do we need to change some of these other tests? We're still using `initialUpdateResponse` above in `assignFromUser` and below.